packages/xchain-bitcoin/src/client.ts

```diff
@@ -249,88 +252,101 @@ class Client implements BitcoinClient, XChainClient {
     return result
   }
 
-  // getBlockTime = async (): Promise<number> => {
-  //   const blocks: Blocks = await getBlocks(this.electrsAPI)
-  //   const times: Array<number> = []
-  //   blocks.forEach((block, index: number) => {
-  //     if (index !== 0) {
-  //       const block1PublishTime = moment.unix(blocks[index - 1].timestamp)
-  //       const block2PublishTime = moment.unix(block.timestamp)
-  //       times.push(block1PublishTime.diff(block2PublishTime, 'seconds'))
-  //     }
-  //   })
-  //   const avgBlockPublishTime = Utils.arrayAverage(times)
-  //   return avgBlockPublishTime
-  // }
-
-  // getTxWeight = async (addressTo: string, memo?: string): Promise<number> => {
-  //   if (!this.validateAddress(addressTo)) {
-  //     throw new Error('Invalid address')
-  //   }
-  //   const network = this.getNetwork(this.net)
-  //   const btcKeys = this.getBtcKeys(this.net, this.phrase)
-  //   const balance = this.getBalance()
-  //   const balancePlaceholder = balance - Utils.dustThreshold - 1
-  //   const psbt = new Bitcoin.Psbt({ network: network }) // Network-specific
-  //   this.utxos.forEach((UTXO) =>
-  //     psbt.addInput({
-  //       hash: UTXO.hash,
-  //       index: UTXO.index,
-  //       witnessUtxo: UTXO.witnessUtxo,
-  //     }),
-  //   )
-  //   psbt.addOutput({ address: addressTo, value: balancePlaceholder }) // Add output
-  //   psbt.addOutput({ address: this.getAddress(), value: 1 }) // change output
-  //   if (memo) {
-  //     const data = Buffer.from(memo, 'utf8') // converts MEMO to buffer
-  //     const OP_RETURN = Bitcoin.script.compile([Bitcoin.opcodes.OP_RETURN, data]) // Compile OP_RETURN script
-  //     psbt.addOutput({ script: OP_RETURN, value: 0 }) // Add OP_RETURN {script, value}
-  //   }
-  //   psbt.signAllInputs(btcKeys) // Sign all inputs
-  //   const tx = psbt.finalizeAllInputs().extractTransaction() // Finalise inputs, extract tx
-  //   const inputs = this.utxos.length // Add weight for each input sig
-  //   return tx.virtualSize() + inputs
-  // }
+  /**
+   * Calculates fees based on fee rate and memo
+   */
+  calcFee = (feeRate: FeeRate, memo?: string): BaseAmount => {
```

Comment:
just as a suggestion:
maybe there is a sense to mark internal things as `private` to decrease public API ?

Reply:
Good point!